[DOCS] Update Learn data pipeline tutorial page (#10828)
great-expectations/great_expectations: docs/docusaurus/docs/reference/learn/integrations/data_pipeline_tutorial.md

---
sidebar_label: 'Integrate GX in a data pipeline'
title: 'Integrate GX in a data pipeline'
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import Admonition from '@theme/Admonition';
import AirflowIcon from "../../../../static/img/airflow_icon.svg";

This tutorial provides working, hands-on examples of how to use GX to add data validation within a data pipeline. The tutorial is hosted on GitHub.

> The tutorial provides a functioning deployment of GX, Airflow, JupyterLab, and Postgres that can be run locally using Docker compose. Educational content is provided as JupyterLab notebooks that can be interactively run and modified to learn and explore how GX integrates with a data pipeline. Airflow is the orchestrator used for the tutorial environment, however, the examples can be adapted to any orchestrator that supports Python code.

Reviewer: I worry a bit that the phrasing of "the examples can be adapted to any orchestrator that supports Python code." could be read as conflicting with the [integration support policy](https://docs.greatexpectations.io/docs/application_integration_support#integrations) which says "Although only Airflow is supported, GX Cloud and GX Core should work with any orchestrator that executes Python code." (the potential conflict being that "can be adapted" sounds definitive while "should work" hedges)

I also wonder if we need a similar caveat for Postgres so folks don't think that's the only supported data source. 

Here's a suggestion that tries to address both of these concerns while future proofing that we may officially support other orchestrators in the future. 

```suggestion
The tutorial provides a functioning deployment of GX, Airflow, JupyterLab, and Postgres that can be run locally using Docker compose. Educational content is provided as JupyterLab notebooks that can be interactively run and modified to learn and explore how GX integrates with a data pipeline. The tutorial environment uses Airflow as the orchestrator and Postgres as a Data Source, but the examples can be adapted to any other [supported integration](/docs/application_integration_support).
```

Author: Great suggestion, I'll incorporate your wording.

> <Admonition type="cta" title="Access this tutorial on GitHub" icon={<AirflowIcon/>}>

Reviewer: This cta still has the airflow icon, I wonder if we should change that too
<img width="657" alt="Screenshot 2025-01-08 at 7 46 46 AM" src="https://github.com/user-attachments/assets/2d1f1c79-97bb-4b3d-86da-99c1476eb837" />

It looks like we have a github icon that could make sense here since the cta title is "Access this tutorial on GitHub" 

https://docs.greatexpectations.io/img/github-mark.svg
